chain/chain/src/chain.rs

```diff
@@ -702,7 +755,11 @@ impl<'a> ChainUpdate<'a> {
                 // Add validator proposals for given header.
                 self.runtime_adapter
                     .add_validator_proposals(
-                        if header.prev_hash != self.genesis_hash { header.prev_hash } else { CryptoHash::default() },
+                        if header.prev_hash != self.genesis_hash {
+                            header.prev_hash
+                        } else {
+                            CryptoHash::default()
```

Comment:
Why do we still have `CryptoHash::default`

Reply:
It was because ValidatorManager doesn't know what is genesis hash.
But I think I figured out how to remove this hackery, let me try